examples/CMakeTests.cmake

```diff
@@ -30,8 +30,6 @@ set (test_ex_CLEANFILES
     group.h5
     groups.h5
     hard_link.h5
-    h5_subfiling_default_example.h5
-    h5_subfiling_custom_example.h5
```

Comment:
Do the examples under HDF5Examples not have a "clean files" list anymore?

Reply:
Need to check.